src/app/shell/workshop-details/side-menu/side-menu.component.html

```diff
@@ -3,7 +3,7 @@
  [fxLayoutAlign]="isMobileScreen ? 'none' : 'center start'"
  fxLayoutGap="2rem">
  <ng-container *ngIf="!isMobileScreen">
-   <app-actions *ngIf="(role !== Role.provider)" [role]="role" [workshop]="workshop"></app-actions>
+   <app-actions *ngIf="(role !== Role.provider && role !== Role.techAdmin)" [role]="role" [workshop]="workshop"></app-actions>
```

Comment:
would it be easier to check if role is parent?

Reply:
@dmitryy90
The role should be either parent or unauthorized 
So I think this way is also ok